src/Discord/WebSockets/Event.php

```diff
@@ -38,6 +38,11 @@ abstract class Event
     public const GUILD_MEMBERS_CHUNK = 'GUILD_MEMBERS_CHUNK';
     public const INTERACTION_CREATE = 'INTERACTION_CREATE';
 
+    // Application Command
+    public const APPLICATION_COMMAND_CREATE = 'APPLICATION_COMMAND_CREATE';
+    public const APPLICATION_COMMAND_UPDATE = 'APPLICATION_COMMAND_UPDATE';
+    public const APPLICATION_COMMAND_DELETE = 'APPLICATION_COMMAND_DELETE';
```

Comment:
I can't find these in the Discord docs, are they documented somewhere?

Reply:
So it turns out they aren't meant to be sent to bots: https://github.com/discord/discord-api-docs/issues/3690#issuecomment-902915187

This shouldn't matter since the client should know when a command is modified.

Reply:
I see, I thought it was not working.. I will remove this